src/dlgPackageExporter.cpp

```diff
@@ -1271,3 +1307,62 @@ QString dlgPackageExporter::getActualPath() const
 {
     return mPackagePath.isEmpty() ? QStandardPaths::writableLocation(QStandardPaths::DesktopLocation) : mPackagePath;
 }
+
+
+//Description Class TextEdit
+dlgPackageExporterDescription::dlgPackageExporterDescription(QWidget* pW) : QTextEdit(pW) {}
+dlgPackageExporterDescription::~dlgPackageExporterDescription() {}
+
+bool dlgPackageExporterDescription::canInsertFromMimeData(const QMimeData* source) const
+{
+    if (source->hasUrls()) {
+        return true;
+    }
+    return QTextEdit::canInsertFromMimeData(source);
+}
+
+
+void dlgPackageExporterDescription::insertFromMimeData(const QMimeData* source)
+{
+    dlgPackageExporter* my_parent = static_cast<dlgPackageExporter*>(topLevelWidget());
+    if (source->hasUrls()) {
+        QTextCursor myCursor = textCursor();
+        int oldPos = myCursor.position();
+        // Allows to insert image at cursor position if using copy/paste
+        if (hasFocus()) {
+            myCursor.setPosition(oldPos);
+        } else {
+            setPlainText(my_parent->mPlainDescription);
+        }
+        QStringList accepted_types;
+        accepted_types << "jpeg"
```

Comment:
What about `svg`?

Reply:
:+1:  need to find out which image formats are supported :wink: